app/scripts/models/auth_brokers/web-channel.js

```diff
@@ -103,13 +103,13 @@ define([
       // The slight delay is to allow the functional tests time to bind
       // event handlers before the flow completes.
       var self = this;
-      return p().delay(100).then(_.bind(self.finishOAuthFlow, self, account));
+      return p().delay(100).then(_.bind(self.finishOAuthFlow, self, account, { action: 'signup' }));
```

Comment:
nit: this line wraps in GitHub side by side view, can you split it before the `.delay(100)` or before the `.then`?